apk-builder.sh

```
@@ -0,0 +1,181 @@
+
+OUTPUT_DIR="$SCRIPT_DIR/dist-apk"
+mkdir -p "$OUTPUT_DIR"
+
+# ── Method 1: Bubblewrap (TWA) ──
+build_bubblewrap() {
+  info "Method 1: Building with Bubblewrap (Trusted Web Activity)..."
+  command -v java &>/dev/null || { warn "Java not found — skipping bubblewrap"; return 1; }
+
+  local url="${1:-http://localhost:18923}"
+  local app_name="${2:-Codex Web}"
+  local package_name="${3:-com.codexweb.app}"
```

Shellcheck (0.11.0)

[warning] package_name appears unused. Verify use (or export if used externally).

(SC2034)